e2e/pnpm_workspace_deps/lib-dupes/BUILD.bazel

```diff
@@ -0,0 +1,26 @@
+load("@aspect_rules_js//npm:defs.bzl", "npm_package")
+load("@aspect_rules_js//js:defs.bzl", "js_library")
+load("@npm//:defs.bzl", "npm_link_all_packages")
+
+npm_link_all_packages(name = "node_modules")
+
+# A library with 2 aliases to one simple dependency (package with no dependencies)
+js_library(
+    name = "js_lib_dupes",
+    srcs = ["index.js"],
+    data = [
+        ":node_modules/@aspect-test/c",
```

Comment:
add a comment that the `data` here is intentional and necessary to check for regressions on #1110